javascript/javascript1/week3/calculate-travel-time.js

```diff
@@ -4,16 +4,15 @@ const travelInformation = {
   };
   
 function travelTimes(speed, distance){
-    let timetoTravel = distance / speed;
-    let splitNumber = (timetoTravel.toString().split('.')); 
-    let test = (splitNumber[0] * 60) + +splitNumber[1];
-    let hours = Math.floor(test / 60); 
-    let minute = test % 60;
-    console.log(`${hours} hours and ${minute} minutes`)
+    const timetoTravel = distance / speed;
+    const splitNumber = (timetoTravel.toString().split('.')); 
```

Comment:
This is fine, but I would encourage you that next time try to split integer and decimal using JS number or math methods.